worlds/civ_6/Items.py

```diff
@@ -0,0 +1,296 @@
+from enum import Enum
+from typing import Dict, List, Optional, TYPE_CHECKING, List

```

Comment:
```suggestion
from typing import Dict, List, Optional, TYPE_CHECKING
```
You're importing List twice